server/controllers/AuthorControllers.js

```diff
@@ -0,0 +1,31 @@
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { Author } = models;
+const { responseMessage } = helpers;
+
+/**
+ * @class AuthorControllers
+ */
+export default class AuthorControllers {
+  /**
+   * @description allows admin add an author
+   * @param {object} request express request object
+   * @param {object} response express response object
+   * @returns {json} json
+   * @memberof AuthorControllers
+   */
+  static async addAuthor(request, response) {
+    try {
+      const { fullname } = request.body;
+      const existingAuthor = await Author.findOne({ where: { fullname } });
```

Comment:
Please ensure that this is not case sensitive. if I enter `Esther Bella` it should match 'esther bella'  and fail to create

Reply:
Noted

Reply:
Treated accordingly